components/org.wso2.carbon.identity.oauth.par/src/main/java/org/wso2/carbon/identity/oauth/par/model/OAuthParRequestWrapper.java

```diff
@@ -43,27 +43,23 @@ public class OAuthParRequestWrapper extends HttpServletRequestWrapper {
      * Wraps the request with parameters obtained from the PAR endpoint.
      *
      * @param request HttpServletRequest.
-     * @param parAuthService ParAuthService.
      * @throws OAuthProblemException OAuthProblemException.
      */
-    public OAuthParRequestWrapper(HttpServletRequest request, ParAuthService parAuthService)
+    public OAuthParRequestWrapper(HttpServletRequest request)
             throws OAuthProblemException {
 
         super(request);
 
-        //get only uuid from request_uri
+        // Get only uuid from request_uri.
         String requestUri = request.getParameter(OAuthConstants.OAuth20Params.REQUEST_URI);
         String uuid = requestUri.replaceFirst(ParConstants.REQUEST_URI_PREFIX, "");
```

Comment:
here, should we have a check to see if the request URI is valid?
something like,
```java
if (!requestUri.startsWith(ParConstants.REQUEST_URI_PREFIX)) {
    throw new ParAuthFailureException("Invalid request_uri format");
}
```